src/components/bulk-email-tool/bulk-email-task-manager/test/BulkEmailContentHistory.test.jsx

```diff
@@ -16,6 +16,9 @@ jest.mock('../data/api', () => ({
 
 describe('BulkEmailContentHistory component', () => {
   beforeEach(() => jest.resetModules());
+  beforeAll(async () => {
```

Comment:
I needed to change a little bit of logic around the `setupTest` tools we use. To prevent tests from breaking because of `InltlProvder`, we run a setup function before all the tests to mock certain things out.

Reply:
This does mean I had to touch every test file, but for now we don't have that many.